scripts/webui_streamlit.py

```diff
@@ -71,7 +71,7 @@ def layout():
 	else:
 		st.session_state["GFPGAN_available"] = False
 
-	if os.path.exists(os.path.join(st.session_state["defaults"].general.RealESRGAN_dir, "experiments","pretrained_models", f"{st.session_state['defaults'].general.RealESRGAN_model}.pth")):
+	if os.path.exists(os.path.join(st.session_state["defaults"].general.RealESRGAN_dir, "realesrgan","weights", f"{st.session_state['defaults'].general.RealESRGAN_model}.pth")):
```

Comment:
?? We say to put the model there in the installation instructions, same goes for ldsr and gfpgan. Inconsistent

Reply:
Agreed. Just realised that the directory structure of the realesrgan repo is different but the models are meant to be downloaded separately.